containers/unit-tests/setup.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh -ex
+
+personality="$1"
+
+dependencies="\
+    appstream-util \
+    autoconf \
+    automake \
+    build-essential \
+    chromium-browser \
+    clang python3 \
+    curl \
+    dbus \
+    gcc-multilib \
+    gdb \
+    git \
+    glib-networking \
+    glib-networking-dbgsym\
+    gtk-doc-tools \
+    intltool \
+    libc6-dbg \
+    libfontconfig1 \
+    libglib2.0-0-dbgsym \
+    libglib2.0-dev \
+    libgnutls28-dev \
+    libgudev-1.0-dev \
+    libjavascript-minifier-xs-perl \
+    libjson-glib-dev \
+    libjson-perl \
+    libkeyutils-dev \
+    libkrb5-dev \
+    liblvm2-dev \
+    libnm-glib-dev \
+    libpam0g-dev \
+    libpcp-import1-dev \
+    libpcp-pmda3-dev \
+    libpcp3-dev \
+    libpolkit-agent-1-dev \
+    libpolkit-gobject-1-dev \
+    libssh-4-dbgsym \
+    libssh-dev \
+    libsystemd-dev \
+    nodejs \
+    npm \
+    pkg-config \
+    pyflakes3 \
+    python3-pep8 \
+    ssh \
+    valgrind \
+    xmlto \
+    xsltproc \
+"
+echo "root:pass" | chpasswd
```

Comment:
This is clearly the wrong password *grin*. All images that the cockpit team builds, and all test suites use "foobar".

Reply:
For some reason I really dislike the password `foobar`...

Reply:
FTR: as discussed on IRC, let's not install sudo and a root password in the first place. It's easy enough to attach as root from outside with `docker exec -it -u root <container> bash`.